openapi_python_client/parser/responses.py

```diff
@@ -79,27 +80,30 @@ def empty_response(
     )
 
 
-def response_from_data(
+def response_from_data(  # noqa: PLR0911
     *,
     status_code: HTTPStatus,
     data: Union[oai.Response, oai.Reference],
     schemas: Schemas,
+    responses: Dict[str, Union[oai.Response, oai.Reference]],
```

Comment:
this is a breaking change to this function's signature....What's the general methodology to validate that this is safe (possibly other places that might call this function?) 
 I think we should have good confidence from passing tests but just calling it out in case there's anything additional we need to consider

Reply:
It's validated by the existing unit tests for the repo, which include a full end-to-end test of running the generator on an entire API spec.

Reply:
> building an Endpoint (which is literally the only code path response_from_data is called from

oh okayy cool... this is what I wanted to confirm!